include/antler/project/location.hpp

```diff
@@ -9,13 +9,21 @@
 
 namespace antler::project::location {
 
+/// If location is github URL then strips out `https://github.com/` from URL
+/// @param loc Github location - either full URL or shorthand
+/// @return string_view shorthand version of full github URL
+[[nodiscard]] std::string_view normalize(std::string_view loc);
```

Comment:
I find name `normalize` too ambiguous. I know this name is something used for unicode conversion or other representation format like adding trailing zeros to a number, etc. I would name it `relative_path` or something.

Reply:
I have renamed it to `strip_github_com`